web-server/assets/src/script/skills.js

```diff
@@ -76,7 +76,24 @@ const filterSkillsList = filter => {
   })
 }
 
+const hideEmptyFilters = (filterElements) => {
+  filterElements.each((index, filter) => {
```

Comment:
change this line to `$('.skills-list-filter').each((index, filter) => {` and dont pass in `filterElements`. its okay for this method to know that CSS selector.